src/schema/users.ts

```diff
@@ -756,10 +757,23 @@ export const resolvers: IResolvers<any, Context> = {
         where: { username: In(generatedUsernames) },
         select: ['username'],
       });
+      const disallowHandles = await ctx.getRepository(DisallowHandle).find({
+        where: { value: In(generatedUsernames) },
+        select: ['value'],
+      });
+
+      const disallowedUsernames = [...usernameChecks, ...disallowHandles].map(
+        (handle) => {
+          if (handle instanceof User) {
+            return handle.username;
+          }
+          return handle.value;
+        },
+      );
 
-      for (const usernameCheck in usernameChecks) {
+      for (const usernameCheck in disallowedUsernames) {
         generatedUsernames = generatedUsernames.filter(
-          (item) => item !== usernameChecks[usernameCheck].username,
+          (item) => item !== disallowedUsernames[usernameCheck],
```

Comment:
This should be better?
```js
generatedUsernames
    .filter(item => !disallowedUsernames.includes(item))
```

Reply:
Ah super yes, I kind of worked on the old code, but good point 👍